app/socket-apis/__tests__/get-points-of-ids.js

```diff
@@ -0,0 +1,123 @@
+// https://github.com/EnCiv/civil-pursuit/issues/203
+import { MongoMemoryServer } from 'mongodb-memory-server'
+import { Mongo } from '@enciv/mongo-collections'
+import { ObjectId } from 'mongodb'
+import Points from '../../models/points'
+import getPointsOfIds from '../get-points-of-ids'
+
+let memoryServer
+let db
+
+const synuser = { synuser: { id: 'user1' } }
```

Comment:
instead of 'user1', please use something like '123456'.  It's a sting version of an ObjectId which should be a hex value.  In some cases other's have not stuck to this, but I'm going to start paying more attention. non compliant values may throw errors in the future.